src/SDKs/Automation/Management.Automation/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure Automation management operations including the ability to create, update and delete runbooks and schedules.")]
 
 [assembly: AssemblyVersion("3.0.1.0")]
-[assembly: AssemblyFileVersion("3.3.0.0")]
+[assembly: AssemblyFileVersion("3.8.0-preview")]
```

Comment:
Please rename this to `3.8.0.0` (-preview is not allowed in file versioning)

Reply:
done